quinn-proto/src/connection/mod.rs

```diff
@@ -1062,6 +1068,18 @@ where
         self.path.remote
     }
 
+    /// The local IP address which was used when the peer established
+    /// the connection
+    ///
+    /// This can be different from the address the endpoint is bound to, in case
+    /// the endpoint is bound to a wildcard address like `0.0.0.0` or `::`.
+    ///
+    /// This will return `None` for clients, as well for servers if capturing
```

Comment:
`as well for` seems a bit odd, add an extra `as`? I also think this docstring would be more useful if it explicitly lists supported platforms.

Reply:
Alternatively, `and` might be a bit more concise.

Reply:
I went for listing the supported platform[s]